File: Data Structures & Algorithms/concatenation-of-array/submission-0.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        var len = nums.Length;
        int[] num = new int[len*2];
        for(int i=0; i<len; i++ )
        {
            num[i]=nums[i];
            num[len+i]=nums[i];
        }
        return num;
    }
}

File: Data Structures & Algorithms/sort-an-array/submission-0.java
class Solution {
    public int[] sortArray(int[] nums) {
        for(int i=0;i<nums.length;i++)
        {
            int num = nums[i]; 
            for(int j=i+1;j<nums.length;j++)
            {
                if(num>nums[j])
                {
                    nums[i] = nums[i]+nums[j];
                    nums[j] = nums[i]-nums[j];
                    nums[i] = nums[i]-nums[j];
                }
                num=nums[i];
            }
        }
        return nums;
    }
}